addons/hr_holidays/static/src/avatar_card_popover_patch.xml

```diff
@@ -7,7 +7,7 @@
             <i t-elif="user.im_status === 'leave_busy'" class="fa fa-fw fa-plane text-danger" title="Busy" role="img" aria-label="User is busy"/>
             <i t-elif="user.im_status === 'leave_offline'" class="fa fa-fw fa-plane text-500" title="Out of office" role="img" aria-label="User is out of office"/>
         </xpath>
-        <xpath expr="//span[@t-esc='user.name']" position="after">
+        <xpath expr="//span[@t-esc='name']" position="after">
```

Comment:
This selector is really weak, I'd much prefer if it targeted a class name (even if introduced just for that) on the specific element or a more descriptive (data-)attribute.